handsontable/src/plugins/trimRows/trimRows.js

```diff
@@ -49,6 +50,46 @@ export const PLUGIN_PRIORITY = 330;
  * // rerender table to see the changes
  * hot.render();
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * const hotRef = useRef();
+ *
+ * ...
+ *
+ * <HotTable
+ *   ref={hotRef}
+ *   data={getData()}
+ *   // hide selected rows on table initialization
+ *   trimRows={[1, 2, 5]}
+ * />
+ *
+ * // access the trimRows plugin instance
+ * const trimRowsPlugin = hotRef.getPlugin('trimRows');
```

Comment:
Same as above - you'll need to call the API methods from the `.current.hotInstance` property of the referenced component.

Reply:
Changed within: https://github.com/handsontable/handsontable/pull/9791/commits/8464a8efd8e87971f4d593025622e98b0e8b20ae